src/main/java/org/truffleruby/core/CoreLibrary.java

```diff
@@ -651,16 +651,18 @@ private void initializeGlobalVariables() {
 
         loadPathStorage = globals.put("$LOAD_PATH",
                 Layouts.ARRAY.createArray(arrayFactory, null, 0));
-        globals.alias("$:", loadPathStorage);
+        globals.alias("$LOAD_PATH", "$:");
+        globals.alias("$LOAD_PATH", "$-I");
```

Comment:
Could we write these in Ruby instead? In order to have less Java code.

Reply:
I guess, but it seems nice to have global definitions together and not spread across files.